centos.org/jobs/foreman-pipelines.yml

```diff
@@ -41,6 +41,26 @@
       - foreman
       - plugins
 
+- project:
+    name: foreman-release-test-32
+    jobs:
+      - 'foreman-pipeline-{type}-{version}-{distro}-{action}'
+    empty: ''
+    distro:
+      - centos7
+      - centos8
+      - centos8-stream
+      - debian10
```

Comment:
nightly has debian11.as well.  And the 3.2.groovy attached to this PR also has debian11.  I think we need to also add debian11 here.